packages/gestalt/src/AlertModal.js

```diff
@@ -0,0 +1,202 @@
+// @flow strict
+import { type Node } from 'react';
+import Box from './Box.js';
+
+import Button from './Button.js';
+import Flex from './Flex.js';
+import Heading from './Heading.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Modal from './Modal.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
```

Comment:
Sorry, I should've been more clear. You'll have to inline the type itself, otherwise we see this:

![Screen Shot 2022-10-12 at 4 55 36 PM](https://user-images.githubusercontent.com/12059539/195468722-cf69ab59-1705-488a-8700-3ec739dab83d.png)

Check out other components using autogen props tables (like NumberField) to see examples